Домашнее задание спринт 8

Nataliya2288 opened the pull request.

Nataliya2288 committed:
Домашнее задание спринт 8

faritowich commented on app/src/main/res/layout/activity_library2.xml:
Для чего в проекте столько лишних файлов? Их всех стоит удалить

faritowich commented on app/src/main/java/com/practicum/playlistmaker/MainActivity.kt:
```diff
@@ -1,11 +1,55 @@
+
+    private fun startActivity(intent: Unit) {
+
+    }
```
Пустой метод

faritowich commented on app/src/main/java/com/practicum/playlistmaker/MainActivity.kt:
```diff
@@ -1,11 +1,55 @@
+
+    }
+
+    private fun Intent(onClickListener: View.OnClickListener, java: Class<SearchActivity>) {
```
Аналогично

faritowich commented on app/src/main/java/com/practicum/playlistmaker/MainActivity.kt:
```diff
@@ -1,11 +1,55 @@
+            // Добавьте код для перехода на экран настроек
+            val intent = Intent(this, SettingActivity::class.java)
+            startActivity(intent)
+            finish() // Завершить текущий экран
```
Для чего finish()? Когда пользователь переходит на след экран, он ожидает что сможет вернуться на предыдущий нажатием на кнопку Назад

Nataliya2288 committed:
Домашнее задание спринт 8 (исправления)
1. Поправила навигацию между экранами
2. Убрала finish()
3. Заменила иконку "поддеожка/suport"- теперь не должно крашиться
4. проверила на эмуляторе- все работает
5. почистила лишние файлы

Nataliya2288 committed:
Домашнее задание спринт 8 (исправления)
1. Поправила навигацию между экранами
2. Убрала finish()
3. Заменила иконку "поддеожка/suport"- теперь не должно крашиться
4. проверила на эмуляторе- все работает
5. почистила лишние файлы
